### DISCUSSION TITLE
Dont have add function in PRBMathSD59x18

### DISCUSSION THREAD

**Discussion by snehsaurav** [PaulRBerg/prb-math#94] [Q&A] [answered]

**Comment by snehsaurav** [comment]
May I know how to do addition using fixed point numbers ?

**Comment by PaulRBerg** [comment, marked as answer]
There's nothing special about adding numbers in PRBMath V2. You can just use the usual `+` operator (don't forget though that by default Solidity implements [checked arithmetic](https://docs.soliditylang.org/en/latest/control-structures.html#checked-or-unchecked-arithmetic), which you can disable via `unchecked`).
